Description: Propagate HDF5 flags to various places
Bug-Debian: https://bugs.debian.org/756689
Forwarded: no
Author: Gilles Filippini <pini@debian.org>
Reviewed-by: Sébastien Villemot <sebastien@debian.org>
Reviewed-by: Rafael Laboissière <rafael@debian.org>
Last-Update: 2016-11-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/libgui/src/module.mk
+++ b/libgui/src/module.mk
@@ -215,6 +215,7 @@ src_libgui_src_la_CPPFLAGS = \
   $(FONTCONFIG_CPPFLAGS) \
   @OCTGUI_DLL_DEFS@ \
   @QT_CPPFLAGS@ \
+  @HDF5_CPPFLAGS@ \
   -I$(srcdir)/libgui/qterminal/libqterminal \
   -Ilibgui/src -I$(srcdir)/libgui/src \
   -I$(srcdir)/libgui/src/m-editor \
--- a/libinterp/module.mk
+++ b/libinterp/module.mk
@@ -6,6 +6,7 @@ include $(top_srcdir)/build-aux/common.m
 
 ## Search local directories before those specified by the user.
 libinterp_liboctinterp_la_CPPFLAGS = \
+  @HDF5_CPPFLAGS@ \
   @OCTINTERP_DLL_DEFS@ \
   -Iliboctave -I$(srcdir)/liboctave \
   -I$(srcdir)/liboctave/cruft/misc \
